src/main.js

```diff
@@ -19,6 +19,8 @@ import Payment from './pages/Payment.vue';
 import Login from './pages/Login.vue';
 import User from './pages/User.vue';
 
+console.log(BUILDDATE + ' ' + VERSION + ' ' + BRANCH);
+
```

Comment:
Yup! deliberately; that's a bit that a tester can check to make sure they're on the right build on thier device (if it's hooked up to a desktop with chrome); Unless there is a better place for a version number in the UI ?